README.md

```diff
@@ -158,3 +168,7 @@ $ npm test
 [connect]: https://stripe.com/connect
 [https-proxy-agent]: https://github.com/TooTallNate/node-https-proxy-agent
 [stripe-js]: https://stripe.com/docs/stripe.js
+
+<!--
+# vim: set tw=79:
+-->
```

Comment:
This doesn't 100% belong with the rest of the this branch, but I was adding them in some of the other libraries, and just put it in here while I was at it.